demo/checkcustomer-rf.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Library                   Collections
+Library                   OperatingSystem
+Library                   SeleniumLibrary
+Library                   ExcelLibrary.py
+Suite Teardown            Close All Browsers
+
+*** Variables ***
+${BROWSER}                gc
+${location}               http://localhost:8080/
+${input_xlsx}             input.xlsx
+${output_xlsx}            output.xlsx
+
+*** Test Cases ***
+Check CustomerIds
+    Log    Start
+    RemoveFile                   ${output_xlsx}
+    OpenBrowser                  ${location}            ${BROWSER}
+    WaitUntilPageContains        Check CustomerId
+
+    # Read input
+    OpenExcelDocument            filename=${input_xlsx}   doc_id=input
+    ${col}=                      Read Excel Column        col_num=1
+    Remove From List             ${col}                   0
+    CloseAllExcelDocuments
+
+    # Create output Excel
+    CreateExcelDocument          output
+    WriteExcelCell               1            1     CustomerId
```

Comment:
Whitespace in this file looks inconsistent. Either align everything nicely or stick to same amount of whitespace with keyword parameters.